packages/api/e2e/api.spec.ts

```diff
@@ -1,33 +1,101 @@
+import { DirectSecp256k1HdWallet } from '@cosmjs/proto-signing';
 import { RegenApi } from '../src/api';
 import { QueryClientImpl } from '../src/generated/cosmos/bank/v1beta1/query';
+import { MsgSend } from '../src/generated/cosmos/bank/v1beta1/tx';
 import { ServiceClientImpl } from '../src/generated/cosmos/tx/v1beta1/service';
 
+const TEST_ADDRESS = 'regen1df675r9vnf7pdedn4sf26svdsem3ugavgxmy46';
+const REDWOOD_NODE_TM_URL = 'http://209.182.218.23:26657';
+const TEST_FEE = {
+  amount: [
+    {
+      denom: 'uregen',
+      amount: '5000',
+    },
+  ],
+  gas: '200000',
+};
+const TEST_MEMO = 'memo';
```

Comment:
We could consider making this memo 'unit test' or 'integration test'. It could leave a living history of our test runs on redwood. Maybe even include the current version number of `regen-js`. Just a thought.

Reply:
good point, "regen-js ${versionNumber} tests" sounds good to me